docs/releases/4.2.10.txt

```diff
@@ -0,0 +1,13 @@
+==========================
+Django 4.2.10 release notes
+==========================
+
+*...*
+
+Django 4.2.10 fixes a bug in 4.2.
+
+Bugfixes
+========
+
+* Fixed a regression in Django 4.2 where migrate prune command should only
+  prune migration files for the specified app (:ticket:`35122`).
```

Comment:
It doesn't qualify for a backport based on our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions). Please remove release notes.